src/renderers/common/Textures.js

```diff
@@ -335,9 +335,7 @@ class Textures extends DataMap {
 
 	needsMipmaps( texture ) {
 
-		if ( this.isEnvironmentTexture( texture ) ) return true;
-
-		return ( texture.isCompressedTexture === true ) || ( ( texture.minFilter !== NearestFilter ) && ( texture.minFilter !== LinearFilter ) );
+		return this.isEnvironmentTexture( texture ) || texture.isCompressedTexture === true || texture.generateMipmaps;
```

Comment:
Including `texture.generateMipmaps` here allows us to simplify the following line:

https://github.com/mrdoob/three.js/blob/20ce0ce14f2138d7b5c9825b8f4a4ad6e9fbfe31/src/renderers/common/Bindings.js#L180

`texture.generateMipmaps === true && ` can be removed.